packages/itwinui-css/backstop/tests/select.html

```diff
@@ -99,6 +99,38 @@ <h2>Default</h2>
         </div>
         <div class="iui-message">Help message.</div>
       </label>
+      <label class="iui-input-container iui-with-message">
+        <div class="iui-label">ComboBox multiselect label</div>
+        <div class="iui-input-with-icon">
+          <input
+            class="iui-input"
+            placeholder="Placeholder…"
+            style="padding-left: min(414px, calc(70% + 20px))"
```

Comment:
I'm just wondering, how would we set this padding in React? Like, how would we know the width of `.iui-select-tag-container`?

Reply:
In react I'm using `useContainerWidth` hook:
![image](https://user-images.githubusercontent.com/83585998/203283981-f4a1cd56-326d-4417-837f-636457676e42.png)

Adding ref object to multiple tags container:
![image](https://user-images.githubusercontent.com/83585998/203284120-e097560a-6a43-47eb-beab-9432f7ef6b60.png)

And using width object for input padding:
![image](https://user-images.githubusercontent.com/83585998/203284237-c72ce8b3-d00c-4c3a-ae28-2c98f4bbcb26.png)